NeuroID/src/main/java/com/neuroid/tracker/service/NIDJobServiceManager.kt

```diff
@@ -133,7 +132,7 @@ internal class NIDJobServiceManager(
     private fun createSendCadenceServer(): Job {
         return CoroutineScope(dispatcher).launch {
             while (userActive && isActive) {
-                delay(5000L)
+                delay(NeuroID.nidSDKConfig.eventQueueFlushInterval * 1000L)
```

Comment:
This would be 10*1000L correct?

Reply:
yeah. ill update the default config to 5.